### DISCUSSION TITLE
proxy settings

### DISCUSSION THREAD

**Discussion by masterman555** [KRTirtho/spotube#1099] [Ideas]
Hello,

it would be nice to have manual proxy settings. it does not seem like the app uses system proxy (it only loads when i force a vpn rather than proxy) and even if it does, manual proxy settings would be a huge quality of life upgrade.

Especially for users in countries with restricted access like Iran and China.

**Comment by nepalivai** [comment]
There is on the setting.
<img width="767" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/50c67e88-8db8-46ac-8cc3-87a3474fcbe2">

Is that it or am i missing something?

**Reply by KRTirtho** [reply]
Nope. It's different.
